packages/syft/src/syft/ast/klass.py

```diff
@@ -129,8 +129,7 @@ def run_class_smpc_method(
             raise ValueError(
                 "There should be a `seed_id_locations` kwargs when doing an operation for MPCTensor"
             )
-
-        kwargs["seed_id_locations"] = str(seed_id_locations)
```

Comment:
Q: Wouldn't it be better to enforce it on `run_class_smpc_method` function? (like having it as a parameter?)

Reply:
I think I enforced it on `run_class_smpc_method` , https://github.com/OpenMined/PySyft/blob/3e6f13584ef3c18c02ebde2b38a7a4ef6cf2cdb9/packages/syft/src/syft/ast/klass.py#L170 or do you mean any other method.